release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## August 15, 2019
+
+The following updates were made in this release:
+
+**Android Core 1.4.3**
+
+* Fixed a bug where the database operation may fail on Android Q.
+* Fixed a crash happening on Android 8/8.1 related to Android's TimeZoneNamesImpl.
+* Other internal bug fixes.
```

Comment:
I don't think we need to mention internal bug fixes in the release notes, right?